lib/spack/spack/test/config.py

```diff
@@ -1132,3 +1132,14 @@ def test_single_file_scope_cache_clearing(env_yaml):
     after = scope.get_section('config')
     assert after
     assert before == after
+
+
+def test_internal_config_cache():
```

Comment:
I'd suggest:
```suggestion
def test_internal_config_scope_cache_clearing():
```
to be consistent with the test above